deblur/workflow.py

```diff
@@ -783,29 +790,33 @@ def launch_workflow(seqs_fp, working_dir, mean_error, error_dist,
                      output_fp=output_derep_fp,
                      min_size=min_size, threads=threads_per_sample)
     # Step 3: Remove artifacts
-    output_artif_fp = remove_artifacts_seqs(seqs_fp=output_derep_fp,
-                                            ref_fp=ref_fp,
-                                            working_dir=working_dir,
-                                            ref_db_fp=ref_db_fp,
-                                            negate=True,
-                                            threads=threads_per_sample,
-                                            sim_thresh=sim_thresh)
+    output_artif_fp, num_seqs_left = remove_artifacts_seqs(seqs_fp=output_derep_fp,
+                                                           ref_fp=ref_fp,
```

Comment:
See [here](https://github.com/biocore/deblur/pull/131/files#diff-352208b099fc0ad645c237f32699335bR794) compared to [here](https://github.com/biocore/deblur/pull/131/files#diff-bd2728968a097d66eb37926bfb6ba6fcR316)